src/main/java/com/example/demo/cart/service/LineItemService.java

```diff
@@ -32,16 +33,14 @@ public void addOrUpdateItem(CartDto.LineItemDto lineItemDto) {
 
         if (existingItem.isPresent()) {
             LineItem item = existingItem.get();
-            item.setQuantity(item.getQuantity() + lineItemDto.quantity());
-            item.setTotalPrice(item.getUnitPrice() * item.getQuantity());
+            item.addQuantity(item.getQuantity());
```

Comment:
지금 코드는 수량을 2배로 늘리는데, 아마 이 코드를 의도하셨을 것 같습니다.

```suggestion
            item.addQuantity(lineItemDto.quantity());
```